larq_zoo/quicknet.py

```diff
@@ -0,0 +1,170 @@
+from typing import Optional, Sequence
+
+import larq as lq
+import tensorflow as tf
+from zookeeper import Field, factory
+
+from larq_zoo import utils
+from larq_zoo.model_factory import ModelFactory
+
+
+@factory
+class QuickNetFactory(ModelFactory):
+
+    num_layers: int = Field(15)
+    initial_filters: int = Field(64)
+
+    input_quantizer = Field(lambda: lq.quantizers.SteSign(clip_value=1.25))
+    kernel_quantizer = Field(lambda: lq.quantizers.SteSign(clip_value=1.25))
+    kernel_constraint = Field(lambda: lq.constraints.WeightClip(clip_value=1.25))
+
+    @property
+    def spec(self):
+        spec = {
```

Comment:
Do we plan to add further specs?

Reply:
I do not add to plan anymore as the large model uses SE